<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="chenyuhuide-MacBook-Pro.local" name="testSuitesLoginPage.FacebookLoginTest" tests="2" failures="0" timestamp="25 四月 2019 08:13:03 GMT" time="0.006" errors="1">
  <testcase name="setUp" time="0.006" classname="testSuitesLoginPage.FacebookLoginTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.io.StringReader.<init>(StringReader.java:50)
at org.jsoup.parser.Parser.parse(Parser.java:107)
at org.jsoup.Jsoup.parse(Jsoup.java:58)
at com.relevantcodes.extentreports.converters.TestConverter.<init>(TestConverter.java:144)
at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:92)
at com.relevantcodes.extentreports.ExtentReports.<init>(ExtentReports.java:375)
at utilities.ExtentFactory.getInstance(ExtentFactory.java:9)
at testSuitesLoginPage.LoginPageTest.setUp(LoginPageTest.java:44)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase name="facebookLoginTest" time="0.000" classname="testSuitesLoginPage.FacebookLoginTest">
    <skipped/>
  </testcase> <!-- facebookLoginTest -->
</testsuite> <!-- testSuitesLoginPage.FacebookLoginTest -->
